src/state/actionCreators/authAC.js

```diff
@@ -1,33 +1,34 @@
-import { apiGetAuth, apiSetAuth } from "../../api";
+import { apiGetAuth, apiSetAuth } from '../../api';
 
 const login = (credentials) => {
-  return async (dispatch) => {
-    let response = await apiSetAuth(credentials);
-    const { token } = response.data;
-    window.localStorage.setItem("token", token);
-    response = await apiGetAuth(token);
-    console.log(response.data);
-    const auth = response.data;
-    dispatch({ type: "SET_AUTH", auth: auth });
-  };
+    return async (dispatch) => {
+        let response = await apiSetAuth(credentials);
+        const { token } = response.data;
+        window.localStorage.setItem('token', token);
+        response = await apiGetAuth(token);
+        console.log(response.data);
+        const auth = response.data;
+        dispatch({ type: 'SET_AUTH', auth: auth });
+    };
 };
 
 const logout = () => {
-  return (dispatch) => {
-    window.localStorage.removeItem("token");
-    dispatch({ auth: {}, type: "SET_AUTH" });
-  };
+    console.log(1);
```

Comment:
Do we need this console log here? If so should we change it from 1 to "Logout".

Reply:
Oh we dont, I was just testing